### PR TITLE
Fix C347874 Overdue fine, returned triggers

### PR DESCRIPTION
**Pull request by re-jas**
https://issues.folio.org/browse/FAT-4430
https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/1012/allure/#suites

### CODE REVIEW

File: cypress/e2e/fee-fine/overdue-fine-returned-triggers.cy.js [hunk 1]
```diff
@@ -348,7 +348,7 @@ describe('Overdue fine', () => {
 
       // need to wait so item can get overdue status
       // eslint-disable-next-line cypress/no-unnecessary-waiting
-      cy.wait(100000);
+      cy.wait(120000);
       cy.visit(TopMenu.checkInPath);
       CheckInActions.checkInItem(itemData.barcode);
       CheckInActions.verifyLastCheckInItem(itemData.barcode);
```
Comment (TetianaParanich) [inline cypress/e2e/fee-fine/overdue-fine-returned-triggers.cy.js:354]: This test has issue like 'Expected to return a value at the end of arrow function'. Please fix it.

**Comment by penfriend** [comment]
please add a link to jenkins build with this autotest

File: cypress/e2e/fee-fine/overdue-fine-returned-triggers.cy.js (hunk 1 above)
Comment (re-jas) [reply cypress/e2e/fee-fine/overdue-fine-returned-triggers.cy.js:355]: done

**Review by TetianaParanich — APPROVED** [review]

File: cypress/e2e/recieve-notice/recieve-notice-checkin.cy.js [hunk 2]
```diff
@@ -79,7 +79,11 @@ describe('Recieving notice: Checkout', () => {
       ServicePoints.createViaApi(testData.userServicePoint);
       testData.defaultLocation = Location.getDefaultLocation(testData.userServicePoint.id);
       Location.createViaApi(testData.defaultLocation);
-      cy.getLoanTypes({ limit: 1 }).then((res) => { testData.loanTypeId = res[0].id; });
```
Comment (IhorBohdan) [inline cypress/e2e/recieve-notice/recieve-notice-checkin.cy.js:82]: method cy.getLoanTypes affected 26 files.Please review again.Does need it to delete?

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**